### DISCUSSION TITLE
How to delete a ContentType

### DISCUSSION THREAD

**Discussion by HengzheLi** [OrchardCMS/OrchardCore#7462] [Q&A] [answered]
It can not be found a "Delete" button in ContentType list page in Admin dashboard. 

Can a ContentType be deleted?

**Comment by hishamco** [comment]
> Can a ContentType be deleted?

Yes

> It can not be found a "Delete" button in ContentType list page in Admin dashboard.

Really?!! perhaps avoid crash the app when content items of a type is exist or we forgot to add it

/cc @agriffard

**Comment by agriffard** [comment]
See #6751

**Reply by hishamco** [reply]
That's expected

**Comment by deanmarcussen** [comment, marked as answer]
There is no delete button in the list, there is a delete button when you select Edit
